lib/installHelpers.js

```diff
@@ -0,0 +1,537 @@
+var _ = require('underscore');
+var async = require('async');
+var chalk = require('chalk');
+var exec = require('child_process').exec;
+var fs = require('fs-extra');
+var path = require('path');
+var prompt = require('prompt');
+var readline = require('readline');
+var request = require('request');
+var semver = require('semver');
+var Spinner = require('cli-spinner').Spinner;
+
+var configuration = require('./configuration');
+var pkg = fs.readJSONSync(path.join(__dirname, '..', 'package.json'));
+
+var DEFAULT_USER_AGENT = 'Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36';
+var DEFAULT_GITHUB_ORG = 'adaptlearning'; // used to pull releases from
+var DEFAULT_SERVER_REPO = `https://github.com/${DEFAULT_GITHUB_ORG}/adapt_authoring.git`;
+var DEFAULT_FRAMEWORK_REPO = `https://github.com/${DEFAULT_GITHUB_ORG}/adapt_framework.git`;
+var REMOTE_NAME = 'adapt-origin';
```

Comment:
This should be `origin otherwise you would have two remotes and people using custom reports would have different git behaviour to those not.

Reply:
My thinking here was that we shouldn't overwrite any existing remote URLs -- I know a lot of people do use origin as a remote name (thought it less likely that adapt-origin is used). Happy to change this though if everyone's ok with the side-effects.

What do you mean by custom reports and different git behaviour?

Reply:
Sorry reports was a typo I meant repos :)

For us, the main use of this code is deploying updates to servers and it seems silly to have multiple remotes there especially as your likely to have `origin` and `adapt-origin` pointing to the same remote if thats where you cloned from initialy. Which is the mostly likely use case.

Reply:
Agreed on weekly catch up to rename remote to origin, and add better error handling to warn users with non-standard remote names.